invenio_app_ils/ui/backoffice/src/pages/LoanDetails/state/actions.js

```diff
@@ -5,7 +5,7 @@ import {
   LOAN_ACTION_SUCCESS,
   SET_LOAN_ACTION_ERROR,
 } from './types';
-import { fetchLoan, postLoan } from './api';
+import { fetchRecord, postRecord } from '../../../common/api';
```

Comment:
To avoid the `../../../`, we could just add a `package.json` at the root, for example:
```
{
    "name": "backoffice"
}
```
And then:
`import { fetchRecord, postRecord } from 'backoffice/common/api';`

Reply:
Apart from the `package.json` with `name: backoffice` that exists in the root of backoffice? I tried a couple of things, but wasn't able to use it as an injection point.